core/src/main/scala/org/apache/spark/scheduler/MapStatus.scala

```diff
@@ -123,7 +123,8 @@ private[spark] object MapStatus {
 private[spark] class CompressedMapStatus(
     private[this] var loc: BlockManagerId,
     private[this] var compressedSizes: Array[Byte],
-    private[this] var _mapTaskId: Long)
+    private[this] var _mapTaskId: Long,
+    private[this] var _checksumVal: Long = 0)
```

Comment:
This looks irrelevant to this PR. Could you revert this change, @bozhang2820 ?

Reply:
Sorry about this. Will revert.